src/main/java/org/springframework/data/redis/connection/jedis/JedisConverters.java

```diff
@@ -200,6 +225,21 @@ public static String toString(byte[] source) {
 		return sentinels;
 	}
 
+	/**
+	 * @param clusterNodes
+	 * @return
+	 * @since 1.7
+	 */
+	public static Set<RedisClusterNode> toSetOfRedisClusterNodes(String clusterNodes) {
+
+		if (StringUtils.isEmpty(clusterNodes)) {
+			return Collections.emptySet();
+		}
+
+		String[] lines = clusterNodes.split(System.getProperty("line.separator"));
```

Comment:
The line separator in `CLUSTER NODES` is not platform-dependent, it's always `\n` (https://github.com/antirez/redis/blob/unstable/src/cluster.c#L3747)